android/src/main/java/com/oney/WebRTCModule/UvcCapturer.java

```diff
@@ -0,0 +1,116 @@
+package com.oney.WebRTCModule;
+
+import android.content.Context;
+import android.util.Log;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.jiangdg.ausbc.callback.IPreviewDataCallBack;
+import com.jiangdg.ausbc.camera.CameraUvcStrategy;
+import com.jiangdg.ausbc.camera.bean.CameraRequest;
+
+import org.webrtc.CapturerObserver;
+import org.webrtc.NV21Buffer;
+import org.webrtc.SurfaceTextureHelper;
+import org.webrtc.SurfaceViewRenderer;
+import org.webrtc.VideoCapturer;
+import org.webrtc.VideoFrame;
+
+public class UvcCapturer implements VideoCapturer {
+
+    private static final String TAG = WebRTCModule.TAG;
+
+    private String deviceId;
+
+    private SurfaceTextureHelper surfaceTextureHelper;
+    private CapturerObserver capturerObserver;
+
+    private int UVC_PREVIEW_WIDTH = 1024;
+    private int UVC_PREVIEW_HEIGHT = 768;
```

Comment:
This is a very weird default resolution. Shall we make it 1280x720 aka 720p ?

Reply:
WebCam resolutions are very different between models, I'll try using 720 instead of 728 and see if it works.

Reply:
It really needs to be 1024x768.

The webcam resolutions are different, like monitor resolution.

logcat
```log
getAllPreviewSizes = [Size(1920x1080@ 0.0,type:4,frame:0,index:0,[]), Size(1024x768@ 0.0,type:4,frame:0,index:1,[]), Size(1024x576@ 0.0,type:4,frame:0,index:2,[]), Size(960x720@ 0.0,type:4,frame:0,index:3,[]), Size(864x480@ 0.0,type:4,frame:0,index:4,[]), Size(800x600@ 0.0,type:4,frame:0,index:5,[]), Size(640x480@ 0.0,type:4,frame:0,index:6,[]), Size(320x240@ 0.0,type:4,frame:0,index:7,[]), Size(1920x1080@ 0.0,type:4,frame:0,index:8,[])]
```

Reply:
Alternatively, figure it out dynamically by querying the supported sizes and picking the one closest to the request resolution that is passed in startCapture, that would make it even better.